src/mmw/js/src/core/utils.js

```diff
@@ -9,6 +9,11 @@ var _ = require('underscore'),
 var M2_IN_KM2 = 1000000;
 var noData = 'No Data';
 
+var SELECT_AREA_PAGE_TITLE = 'Choose Area of Interest';
+var ANALYZE_PAGE_TITLE = 'Geospatial Analysis';
+var COMPARE_PAGE_TITLE = 'Compare';
+var PROJECTS_PAGE_TITLE = 'Projects';
```

Comment:
Consider just setting `selectAreaPageTitle` etc below to these string values, rather than using the intermediate constants.